src/components/Card.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
```

Comment:
We don't need to add this line of code in in React 18.2

Reply:
Done